### PR TITLE
Nazar/feature/user routes

### PR DESCRIPTION
**Pull request by Nazar-kob**

### CODE REVIEW

File: backend/app/config.py [hunk 1]
```diff
@@ -8,7 +6,9 @@ class Settings(BaseSettings):
     JWT_SECRET: str = "<None>"
     ACCESS_TOKEN_EXPIRE_MINUTES: int = 30
     DATABASE_URI: str = ""
```
[the remark this answers is not in the document]
Comment (SvyatoslavArtymovych) [reply backend/app/config.py:8]: The same for JWT_SECRET

File: backend/tasks/db_utils.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+from invoke import task
+from app.config import settings
```
Comment (SvyatoslavArtymovych) [inline backend/tasks/db_utils.py:2]: Sort and split your imports by empty line

File: backend/tests/database.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
+from faker import Faker
+
+
```
Comment (SvyatoslavArtymovych) [inline backend/tests/database.py:3]: Too many blank lines

**Review by SvyatoslavArtymovych — COMMENTED** [review]
Have you run flake8?

**Review by kipod — APPROVED** [review]
Very good.
But take a look on my comments

File: backend/app/router/post.py [hunk 4]
```diff
@@ -1,9 +1,11 @@
 from fastapi import HTTPException, Response, Depends, APIRouter
 from typing import List
-from app import model, oauth2, schema
+from app import model, schema
```
[the remark this answers is not in the document]
Comment (Nazar-kob) [reply backend/app/router/post.py:3]: I got you

**Pull request merged**